src/styles/blocks/catalog.scss

```diff
@@ -5,6 +5,7 @@
   max-width: 944px;
   margin: 0 auto;
   padding: 50px 0;
+  cursor: pointer;
```

Comment:
It's incorrect to add `cursor: pointer;` on the entire catalogue. Just remove it from the `:hover` and add it to the `&__card`.